code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyleTests.kt

```diff
@@ -0,0 +1,310 @@
+            overflow = TextOverflow.Clip,
+            softWrap = true,
+            maxLines = 1,
+            minLines = 1
+        )
+        val buttonRowStyle = AepRowStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            horizontalArrangement = Arrangement.Center,
+            verticalAlignment = Alignment.CenterVertically
+        )
+        val buttonStyle = AepButtonStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            enabled = true,
+            elevation = mock(ButtonElevation::class.java), // using a mock as ButtonElevation requires a composable function
+            shape = CircleShape,
+            border = BorderStroke(1.dp, Color.Red),
+            contentPadding = null,
+            buttonTextStyle = AepTextStyle(
+                modifier = Modifier.size(100.dp, 100.dp),
+                textStyle = TextStyle(color = Color.Red, fontSize = 16.sp),
+                overflow = TextOverflow.Clip,
+                softWrap = true,
+                maxLines = 1,
+                minLines = 1
+            )
+        )
+        val dismissButtonStyle = AepIconStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            contentDescription = "content description",
+            tint = { Color.Blue }
+        )
+        val dismissButtonAlignment = Alignment.BottomEnd
+        val cardStyle = AepCardStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            shape = RectangleShape
+        )
+        val rootRowStyle = AepRowStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            horizontalArrangement = Arrangement.Center,
+            verticalAlignment = Alignment.CenterVertically
+        )
+        val textColumnStyle = AepColumnStyle(
+            modifier = Modifier.size(100.dp, 100.dp),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+        )
+
+        val style = SmallImageUIStyle.Builder()
+            .imageStyle(imageStyle)
+            .titleAepTextStyle(titleTextStyle)
+            .bodyAepTextStyle(bodyTextStyle)
+            .buttonRowStyle(buttonRowStyle)
+            .buttonStyle(arrayOf(buttonStyle, buttonStyle))
+            .dismissButtonStyle(dismissButtonStyle)
+            .dismissButtonAlignment(dismissButtonAlignment)
+            .cardStyle(cardStyle)
+            .rootRowStyle(rootRowStyle)
+            .textColumnStyle(textColumnStyle)
+            .build()
+
+        // verify image style parameters
+        assertEquals(imageStyle.alpha, style.imageStyle.alpha)
+        assertEquals(imageStyle.colorFilter, style.imageStyle.colorFilter)
+        assertEquals(imageStyle.contentDescription, style.imageStyle.contentDescription)
+        assertEquals(imageStyle.alignment, style.imageStyle.alignment)
+        assertEquals(imageStyle.contentScale, style.imageStyle.contentScale)
+        assertEquals(imageStyle.modifier, style.imageStyle.modifier)
```

Comment:
Can you add helper methods for comparing style totally similar elements. That will allow you to re-use them for other templates  and make these tests less verbose (thought partial style overrides will still need to compared individually)